backend/inspirehep/submissions/views.py

```diff
@@ -191,7 +200,34 @@ def load_data_from_request(self):
     def start_workflow_for_submission(self, record):
         record["acquisition_source"] = self.get_acquisition_source()
         payload = {"data": record}
-        return self.send_post_request_to_inspire_next("/workflows/authors", payload)
+
+        if current_app.config.get("FEATURE_FLAG_ENABLE_SEND_TO_BACKOFFICE"):
+            try:
+                payload_backoffice = {
+                    "data": record,
+                    "workflow_type": "AUTHOR_CREATE",
```

Comment:
this function is used for both create an update

Reply:
Okay, besides passing a parameter to `start_workflow_for_submission`  for create/update depending if method POST or PUT is used, is there any way to discern the two?